engine/common/sys_con.c

```diff
@@ -230,6 +230,9 @@ void Sys_PrintLog( const char *pMsg )
 	time( &crt_time );
 	crt_tm = localtime( &crt_time );
 
+	// strip color codes
+	COM_StripColors( pMsg, (char *)pMsg );
```

Comment:
You're modifying the string that you're not supposed to modify. Just a few lines below this, you have Sys_PrintColorized that just converts color codes to escape sequences that terminals interpret as color change.